dashboard/test/models/sections/section_test.rb

```diff
@@ -71,6 +71,78 @@ class SectionTest < ActiveSupport::TestCase
     end
   end
 
+  test 'changing sharing_disabled to true updates user settings' do
+    student = create :student, sharing_disabled: false
+    section = Section.create @default_attrs
```

Comment:
nit: instead of creating the section with `@default_attrs`, why not use the one that already exists: [@section](https://github.com/code-dot-org/code-dot-org/blob/ea6b02539634f8599954570f694dd9388e3b6fe7/dashboard/test/models/sections/section_test.rb#L8)?